Plugin.php

```diff
@@ -151,6 +151,10 @@ protected function extendBrandSettingsData(): void
      */
     protected function extendBrandSettingsForm(): void
     {
+        BrandSetting::extend(function($model) {
+            $model->addAttachOneRelation('backgroundImage', [\System\Models\File::class]);
```

Comment:
Should probably `background_image`, not `backgroundImage`.

Reply:
done